Source/WebCore/style/StyleTreeResolver.cpp

```diff
@@ -1503,12 +1526,19 @@ std::unique_ptr<RenderStyle> TreeResolver::generatePositionOption(const Position
         return rule->properties();
     };
 
-    auto builderFallback = BuilderPositionTryFallback {
-        .properties = resolveFallbackProperties(),
-        .tactics = fallback.tactics
-    };
+    std::optional<BuilderPositionTryFallback> builderFallback;
+    if (fallback) {
+        builderFallback = BuilderPositionTryFallback {
+            .properties = resolveFallbackProperties(*fallback),
+            .tactics = fallback->tactics
+        };
+    }
+
+    auto style = resolveAgainInDifferentContext(resolvedStyle, styleable, *resolutionContext.parentStyle, PropertyCascade::normalPropertyTypes(), WTFMove(builderFallback), resolutionContext);
+    if (style)
+        style->setUsedPositionTryFallback(std::optional<PositionTryFallback> { fallback });
```

Comment:
I don't understand how this works. Why do we set the used position try fallback when generating a position option? Generating an option does not mean we are using it?